test/JDBC/init.sh

```diff
@@ -1,7 +1,7 @@
  
 #create test user and database from psql terminal
 echo "============================== CREATING USER AND DATABASE =============================="
-psql -U "$USER" -d postgres -a << EOF
+sudo ~/postgres/bin/psql -U "$USER" -d postgres -a << EOF
```

Comment:
Was there an issue here? If we change this, then it enforces every developers/github actions to have `~/postgres` directory.

Reply:
Oh sorry, I didn't mean to include this in the commit. I will change it back.